### PR TITLE
ABP-7688 QA changes

### PR DESCRIPTION
**Pull request by raw1697**
PR has came back after testing in QA

### CODE REVIEW

**Comment by raw1697** [comment]
NOTE: welsh translation reads as: "Ar ôl i chi gyflwyno newidiadau, ni fyddwch yn gallu diwygio’r manylion cyswllt eto am 4 wythnos."
Because of mac/intelij encoding the letter 'ô' isn't being decoded correctly, in for to fix this I have replaced it with '&#244;' which is the code for this letter. Is this ok do so?

Link: https://www.compart.com/en/unicode/U+00F4

File: conf/messages [hunk 1]
```diff
@@ -477,8 +477,8 @@ update-contact-details.start-page.body.li4=proprietor
 update-contact-details.start-page.body.li5=partner
 update-contact-details.start-page.body.p2=If your role is on this list, you do not need additional authorisation.
 update-contact-details.start-page.body.h2=Before you start
-update-contact-details.start-page.body.p3=We'll ask you to confirm that you have authorisation before you submit any changes. We'll also need your name and telephone number.
-update-contact-details.start-page.body.p4=In some circumstances we'll ask for your agent codes for Self Assessment and Corporation Tax. The codes were sent by letter when your organisation asked for agent access to those services.
+update-contact-details.start-page.body.p3=We''ll ask you to confirm that you have authorisation before you submit any changes. We''ll also need your name and telephone number.
```
Comment (geoffreywatson) [inline conf/messages:480]: there's a smart quotes (shift + option + ]) - better than double ''

Comment (raw1697) [reply conf/messages:480]: Nice ok

File: conf/messages [hunk 2]
```diff
@@ -477,8 +477,8 @@ update-contact-details.start-page.body.li4=proprietor
 update-contact-details.start-page.body.li5=partner
 update-contact-details.start-page.body.p2=If your role is on this list, you do not need additional authorisation.
 update-contact-details.start-page.body.h2=Before you start
-update-contact-details.start-page.body.p3=We'll ask you to confirm that you have authorisation before you submit any changes. We'll also need your name and telephone number.
-update-contact-details.start-page.body.p4=In some circumstances we'll ask for your agent codes for Self Assessment and Corporation Tax. The codes were sent by letter when your organisation asked for agent access to those services.
+update-contact-details.start-page.body.p3=We?ll ask you to confirm that you have authorisation before you submit any changes. We?ll also need your name and telephone number.
```
Comment (geoffreywatson) [inline conf/messages:480]: what's this? Try shift + option + right square bracket

Comment (raw1697) [reply conf/messages:480]: I did use the smart quotes I'm not sure why it's changed to a '?'

**Review by geoffreywatson — APPROVED** [review]

**Pull request merged**